spec/models/genre_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe Genre do
+  describe 'associations' do
+    it { should have_many :songs }
+  end
+
+  describe 'validations' do
+    it { should validate_presence_of :name }
```

Comment:
I think name should probably be unique.  In which case you'd also want a DB constraint.